bot/commands/userinfo.js

```diff
@@ -0,0 +1,86 @@
+const {MessageEmbed, Permissions} = require('discord.js');
+const utils = require('../utils.js');
+
+module.exports = {
+    active: true,
+    name: 'userinfo',
+    aliases: 'ui',
+    description: (lang) => lang.get('userInfoDescription'),
+    cooldown: 5,
+    categoryID: 1,
+    execute: async (message, args) => {
+        const {channelLanguage} = message;
+        if(
+            message.guild
+            &&
+            !message.guild.me
+                .permissionsIn(message.channel)
+                .has(Permissions.FLAGS.EMBED_LINKS)
+        ) return message.reply(channelLanguage.get('botEmbed'));
+        const id = args ?? args[0]?.match(/^(?:<@)?!?(\d{17,19})>?$/)?.[1];
+
+        const user = id && await message.client.users.fetch(id).catch(() => null) || message.author;
+        const member = user === message.author ? message.member : message.guild?.members?.cache.get(id);
```

Comment:
you should fetch the member to make sure to get it if it exists
```suggestion
        const member = user === message.author ? message.member : await message.guild?.members?.fetch(id).catch(() => null);
```